packages/aws-lambda-otel-extension/opt/otel-extension/external/helper.js

```diff
@@ -10,6 +10,39 @@ const extensionVersion = (() => {
   return require('../version');
 })();
 
+const cleanResponseData = (data) => {
+  let response = '';
+  data.excludedBody = false;
+
+  let responseData = data.responseData;
+  if (typeof responseData === 'string') {
+    try {
+      responseData = JSON.parse(responseData);
+    } catch (error) {
+      responseData = data.responseData;
```

Comment:
I think there's no need for this . if `JSON.parse` fails, `responseData` will remain `data.responseData`

(and for lint purposes we may add `/* noop */` comment)

Reply:
Ahh nice yeah the `/* noop */` this is exactly what I was looking for. Thanks man 👍